### ISSUE TITLE
Make Switch draggable

### ISSUE THREAD

**Issue by thisconnect** [ant-design/ant-design#29289]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
It would be a nice improvement
- ~~accessibility: if the switch was focusable with keyboard~~
- UX: if the switch could be dragged like on iPhone

see react-switch implementation for keyboad focus and drag behavior, note: if you drag the toggle only by a bit, it won't toggle but go back. Only if you drag over 50% then it will go to the other side. 

- Demo https://react-switch.netlify.app/
- NPM https://www.npmjs.com/package/react-switch

### What does the proposed API look like?
no change or optionally similar to react-switch

**Comment by afc163** [comment]
> accessibility: if the switch was focusable with keyboard

We support using keyboad to focus now.

**Comment by thisconnect** [comment]
> We support using keyboad to focus now.

you are right, I didn't see it. In that case this feature request is only about the UX improvement so that "the switch could be dragged like on iPhone". I will update the description
